gitmigration.md

```diff
@@ -0,0 +1,48 @@
+# GITHUB MIGRATION PLAN #
+
+## Objectives ##
+
+-  https://github.com/twisted/twisted will be the "repository of truth" for Twisted.
+     - Twisted releases will be done from GitHub
+     - the Twisted developers who are now "core committers" for SVN, must be 
+       given access to be "core committers" to https://github.com/twisted/twisted
+
+- Trac will still be used for filing tickets.  Filing issues in GitHub will be disabled.  
+
+## Task Items ##
+
+- Task items for this effort will be tracked here: https://github.com/twisted-infra/braid/milestones/Migrate-to-Git
+
+## Plan ##
+### Staging Environment ###
+- [ ] Create a staging repository on GitHub, NOT under ``twisted/twisted``
+- [ ] Do a Git -> SVN format migration -- turning SVN commits into Git commits for ``trunk`` (git-svn does this fine). Investigate trimming everything but tags and trunk/master.
```

Comment:
@tomprince : that item is there because @hawkowl had it in her plan.  I am fine either way with using the existing mirror, or starting from scratch and creating a new repository with git-svn.

It doesn't take that long to create a new git repository with git-svn, so it is no big deal starting from scratch.